test/spec/build.spec.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs');
+var path = require('path');
+var temp = require('temp').track();
+
+var build = require('../../lib/build');
+
+var helper = require('../helper');
+
+var assert = helper.assert;
+var fixtures = path.join(__dirname, '..', 'fixtures');
+
+
+describe('build', function() {
+  it('creates an output file', function(done) {
```

Comment:
You could bump up the timeout here instead of globally (with `this.timeout(30000)`).  This would save 28s of boredom for people who add an async test elsewhere and forget to call `done`.